DBMS/CommandJoin/question1/Create_Script.sql

```diff
@@ -0,0 +1,68 @@
+drop database StoreFront;
+create database StoreFront;
+use StoreFront;
+
+create table User (User_ID varchar(25) NOT NULL PRIMARY KEY, Name varchar(25));
+
+create table Shopper(Shopper_ID varchar(25),FOREIGN KEY (Shopper_ID) 
```

Comment:
For primary keys, we don't need to mention the entity name. For eg, id of shopper table must be named "id" and not "shopper_id". "shopper_id" will be used as the column name when this id is used as a foreign key in other table